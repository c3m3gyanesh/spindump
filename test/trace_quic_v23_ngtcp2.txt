A test between Picoquic and the Ngtcp2 implementation. The following commands were used:

    ./picoquic/picoquicdemo -l ./test.log -L -a "h3-23" -v ff000017 nghttp2.org 4433 /50000
    ./picoquic/picoquicdemo -l ./test.log -L -a "h3-23" -v ff000017 nghttp2.org 4433 /2000
