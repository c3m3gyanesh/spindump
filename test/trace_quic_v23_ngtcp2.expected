QUIC 172.16.114.251 <-> 139.162.123.134 0823f5574f09c959-6276c58288feede7 (56251:4433) at 1571177354341278 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 0823f5574f09c959-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354577163 measurement up right 235885 packets 1 0 bytes 1280 0 bandwidth 1280 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354577753 spinvalue up 0 responder packets 2 3 bytes 1429 3117 bandwidth 1429 3117 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354578042 spinvalue up 1 initiator packets 3 4 bytes 1791 3428 bandwidth 1791 3428 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354578083 spinvalue up 1 initiator packets 4 4 bytes 1911 3428 bandwidth 1911 3428 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354891086 spinvalue up 0 responder packets 5 5 bytes 3379 4025 bandwidth 3379 4025 note "V23,no spin"
UDP 172.16.114.251 <-> 139.162.123.134 56251:4435 at 1571177354891608 new starting packets 0 0 bytes 0 0 note "No response"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354891609 spinvalue up 1 initiator packets 5 6 bytes 3379 5305 bandwidth 3379 5305 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354892158 spinvalue up 0 responder packets 6 6 bytes 3462 5305 bandwidth 3462 5305 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354893522 spinvalue up 0 responder packets 6 7 bytes 3462 6585 bandwidth 3462 6585 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354893799 spinvalue up 1 initiator packets 6 8 bytes 3462 7865 bandwidth 3462 7865 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354893939 spinvalue up 0 responder packets 7 8 bytes 3545 7865 bandwidth 3545 7865 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354895597 spinvalue up 0 responder packets 7 9 bytes 3545 9145 bandwidth 3545 9145 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354895801 spinvalue up 1 initiator packets 7 10 bytes 3545 10425 bandwidth 3545 10425 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354896610 spinvalue up 0 responder packets 8 10 bytes 3628 10425 bandwidth 3628 10425 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354897529 spinvalue up 0 responder packets 8 11 bytes 3628 11705 bandwidth 3628 11705 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354897691 spinvalue up 1 initiator packets 8 12 bytes 3628 12985 bandwidth 3628 12985 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354898417 spinvalue up 0 responder packets 9 12 bytes 3711 12985 bandwidth 3711 12985 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354898868 spinvalue up 0 responder packets 9 13 bytes 3711 14265 bandwidth 3711 14265 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354899052 spinvalue up 1 initiator packets 9 14 bytes 3711 15545 bandwidth 3711 15545 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354899860 spinvalue up 0 responder packets 10 14 bytes 3794 15545 bandwidth 3794 15545 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354900790 spinvalue up 0 responder packets 10 15 bytes 3794 16825 bandwidth 3794 16825 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354901010 spinvalue up 1 initiator packets 10 16 bytes 3794 18105 bandwidth 3794 18105 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354902080 spinvalue up 0 responder packets 11 16 bytes 3877 18105 bandwidth 3877 18105 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354902808 spinvalue up 0 responder packets 11 17 bytes 3877 19165 bandwidth 3877 19165 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177354903013 spinvalue up 1 initiator packets 11 18 bytes 3877 19224 bandwidth 3877 19224 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355111929 spinvalue up 0 responder packets 12 18 bytes 3960 19224 bandwidth 3960 19224 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355112159 spinvalue up 1 initiator packets 12 19 bytes 3960 20504 bandwidth 3960 20504 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355113531 spinvalue up 0 responder packets 13 19 bytes 4043 20504 bandwidth 4043 20504 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355114535 spinvalue up 0 responder packets 13 20 bytes 4043 21784 bandwidth 4043 21784 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355114684 spinvalue up 1 initiator packets 13 21 bytes 4043 22836 bandwidth 4043 22836 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355115399 spinvalue up 0 responder packets 14 21 bytes 4126 22836 bandwidth 4126 22836 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355116358 spinvalue up 0 responder packets 14 22 bytes 4126 24116 bandwidth 4126 24116 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355116509 spinvalue up 1 initiator packets 14 23 bytes 4126 25396 bandwidth 4126 25396 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355117726 spinvalue up 0 responder packets 15 23 bytes 4209 25396 bandwidth 4209 25396 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355119297 spinvalue up 0 responder packets 15 24 bytes 4209 26676 bandwidth 4209 26676 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355119450 spinvalue up 1 initiator packets 15 25 bytes 4209 27956 bandwidth 4209 27956 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355119529 spinvalue up 0 responder packets 16 25 bytes 4292 27956 bandwidth 4292 27956 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355120137 spinvalue up 0 responder packets 16 26 bytes 4292 28015 bandwidth 4292 28015 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355122099 spinvalue up 0 responder packets 16 27 bytes 4292 29295 bandwidth 4292 29295 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355122788 spinvalue up 0 responder packets 16 28 bytes 4292 30575 bandwidth 4292 30575 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355127303 spinvalue up 0 responder packets 16 29 bytes 4292 31855 bandwidth 4292 31855 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355128078 spinvalue up 0 responder packets 16 30 bytes 4292 33135 bandwidth 4292 33135 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355129501 spinvalue up 0 responder packets 16 31 bytes 4292 34415 bandwidth 4292 34415 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355130839 spinvalue up 0 responder packets 16 32 bytes 4292 35695 bandwidth 4292 35695 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355132028 spinvalue up 0 responder packets 16 33 bytes 4292 36975 bandwidth 4292 36975 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355133015 spinvalue up 0 responder packets 16 34 bytes 4292 38255 bandwidth 4292 38255 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355133501 spinvalue up 0 responder packets 16 35 bytes 4292 39535 bandwidth 4292 39535 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355135071 spinvalue up 0 responder packets 16 36 bytes 4292 40815 bandwidth 4292 40815 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355137238 spinvalue up 0 responder packets 16 37 bytes 4292 42095 bandwidth 4292 42095 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355137244 spinvalue up 0 responder packets 16 38 bytes 4292 43375 bandwidth 4292 43375 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355137246 spinvalue up 0 responder packets 16 39 bytes 4292 44655 bandwidth 4292 44655 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355137247 spinvalue up 0 responder packets 16 40 bytes 4292 45935 bandwidth 4292 45935 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355137249 spinvalue up 0 responder packets 16 41 bytes 4292 47215 bandwidth 4292 47215 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355137250 spinvalue up 0 responder packets 16 42 bytes 4292 48495 bandwidth 4292 48495 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355137252 spinvalue up 0 responder packets 16 43 bytes 4292 49775 bandwidth 4292 49775 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355338265 spinvalue up 0 responder packets 16 44 bytes 4292 50615 bandwidth 4292 50615 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355338271 spinvalue up 0 responder packets 16 45 bytes 4292 51895 bandwidth 4292 51895 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355338273 spinvalue up 0 responder packets 16 46 bytes 4292 53175 bandwidth 4292 53175 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355340445 spinvalue up 0 responder packets 16 47 bytes 4292 54455 bandwidth 4292 54455 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 23f5574f09c959df-120b91e6a012df5fd6f94bda07175f8884ef (56251:4433) at 1571177355340452 spinvalue up 0 responder packets 16 48 bytes 4292 55735 bandwidth 4292 55735 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 08ec7a9049d31e97-d7b78426d12d0c9e (54952:4433) at 1571177364459844 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 08ec7a9049d31e97-12260be4fae368517d87bbc9bad8ad3168a4 (54952:4433) at 1571177364710829 measurement up right 250985 packets 1 0 bytes 1280 0 bandwidth 1280 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 ec7a9049d31e973f-12260be4fae368517d87bbc9bad8ad3168a4 (54952:4433) at 1571177364711897 spinvalue up 0 initiator packets 2 1 bytes 1705 510 bandwidth 1705 510 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 ec7a9049d31e973f-12260be4fae368517d87bbc9bad8ad3168a4 (54952:4433) at 1571177364714252 spinvalue up 0 responder packets 3 1 bytes 3173 510 bandwidth 3173 510 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 ec7a9049d31e973f-12260be4fae368517d87bbc9bad8ad3168a4 (54952:4433) at 1571177364714256 spinvalue up 0 responder packets 3 2 bytes 3173 1790 bandwidth 3173 1790 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 ec7a9049d31e973f-12260be4fae368517d87bbc9bad8ad3168a4 (54952:4433) at 1571177364714258 spinvalue up 0 responder packets 3 3 bytes 3173 2961 bandwidth 3173 2961 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 ec7a9049d31e973f-12260be4fae368517d87bbc9bad8ad3168a4 (54952:4433) at 1571177364714692 spinvalue up 1 initiator packets 3 4 bytes 3173 3020 bandwidth 3173 3020 note "V23,no spin"
QUIC 172.16.114.251 <-> 139.162.123.134 ec7a9049d31e973f-12260be4fae368517d87bbc9bad8ad3168a4 (54952:4433) at 1571177365025525 spinvalue up 1 initiator packets 4 5 bytes 3248 3376 bandwidth 3248 3376 note "V23,no R-spin"
